test/performance/odp_sched_pktio.c

```diff
@@ -238,6 +238,24 @@ static int worker_thread_direct(void *arg)
 			test_global->worker_stat[worker_id].rx_pkt += num_pkt;
 	}
 
+	/*
+	 * Free prefetched packets before exiting worker thread as
+	 * such packets can block main thread cleanup cause buffer leak.
```

Comment:
There is a grammar issue in the sentence, making it hard to understand.

Reply:
Ack, will fix in V2.